lib/core/msdf-sys/build.rs

```diff
@@ -0,0 +1,39 @@
+extern crate download_lp;
+
+mod msdfgen_wasm {
+
+    pub const VERSION  : &str = "v1.0";
+    pub const FILENAME : &str = "msdfgen_wasm.js";
+
+    pub fn download() {
+        let url = format!(
+            "https://github.com/luna/msdf/releases/download/{version}/{filename}",
+            version = VERSION,
+            filename = FILENAME
+        );
+
+        if std::path::Path::new(FILENAME).exists() {
+            std::fs::remove_file(FILENAME).unwrap();
+        }
+
+        // Note [Downloading to src dir]
+        download_lp::download(url.as_str(), ".").unwrap();
```

Comment:
I don't like the idea to keep 3rd "binaries" in src folder - maybe we create a third-party folder where such things are downlaoded? We can as well download them to `dist` (or whatever the destination directory is called). After thinking lomnger about it, I'm more and more convinced that we should reuse dist folder here as this is just some kind of building artifact.

Reply:
But the cargo "target" directory is like "dist" directory, and bound js files cannot be downloaded there, as I wrote in note. And the whole "msdf-sys" crate is by convention a wrapper for third party library, so do we really need additional "third-party" directory?

And this file is reused in future builds - the build.rs script is run only on first build or when the script itself changed

Reply:
Oh the note is in fact explaining it. Sorry I misread it earlier. Have you been asking about it on Rust Discord channel ?